.github/workflows/checks.yaml

```diff
@@ -3,19 +3,15 @@
 
 name: Python package
 
-on:
-  push:
-    branches: [ dev ]
-  pull_request:
-    branches: [ dev ]
+on: [push]
```

Comment:
I think you can have the checks run for `pull_request` events as well

Reply:
Done